### ISSUE TITLE
example 'kilo' does not support special characters

### ISSUE THREAD

**Issue by Ryder17z** [jupyter-xeus/cpp-terminal#246]
"Debug Assertion Failed"
points to file "minkernel/crts/ucrt/appcrt/convert/istype.cpp" line 36
"Expression: c > = -1 && c < = 255"

Would be nice to have utf-8 support, or at least utilize the older windows component for supporting non-english characters like 'å'

**Comment by flagarde** [comment]
@Ryder17z I agree this is a problem I'm aware of. I worked a bit on this but still not PR for now. I imagine example are here to show what the library can do and to trigger such problem and propose some changes to the library to fix them.

**Comment by certik** [comment]
I see. The other examples support utf8, but not kilo. We'll have to fix it. Thanks @flagarde for working on it.

**Comment by flagarde** [comment]
@Ryder17z Could you give me more informations : 
1) which line create the crash ?
2) Which text are you writing ?
3) You open a file or write it ?

I guess you are using window$ ?

**Comment by Ryder17z** [comment]
No file open, I just hit 'å' by accident.
The issue is on win10, yes.
The error message is quoted above, with a supposed location of the issue

**Comment by flagarde** [comment]
@Ryder17z By the way, are you using the tag 1.0.0 or the upstream version ?

**Comment by Ryder17z** [comment]
just whatever is  on the master branch
